### PR TITLE
[#109] Add `--path` option to the `default-config` command to print default config location instead

### LINKED ISSUE #109: Implement the '--path' option for the 'default-config' command
Instead of outputting the config content, it should output the path to the default config location.

Relevant code parts:

* CLI argument type

https://github.com/chshersh/tool-sync/blob/df7d4768c9cff2a91616bd9686bc928a52fc8c10/src/config/cli.rs#L21-L22

* Running the command

https://github.com/chshersh/tool-sync/blob/df7d4768c9cff2a91616bd9686bc928a52fc8c10/src/lib.rs#L21

* Creating the path

https://github.com/chshersh/tool-sync/blob/df7d4768c9cff2a91616bd9686bc928a52fc8c10/src/lib.rs#L29-L44

### PR DESCRIPTION
**Pull request by zixuanzhang-x**
Resolves #109 

### Additional tasks

- [x] Documentation for changes provided/changed
- [ ] Tests added
- [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by zixuanzhang-x** [comment]
Here are some example outputs:
- `tool default-config -h`
  Screen Shot 2022-10-03 at 9 00 15 PM

- `tool default-config --path`
   Screen Shot 2022-10-03 at 9 01 12 PM


- `tool default-config`
  <details>
    <summary>Large photo, click to expand</summary>
    Screen Shot 2022-10-03 at 9 00 41 PM
   </details>

File: README.md [hunk 1]
```diff
@@ -244,6 +244,12 @@ Print the default config to stdout:
 tool default-config
 ```
 
+Print the default config location to stdout:
+
+```shell
+tool default-config --path
+```
```
Comment (chshersh) [inline README.md:251]: Thanks a lot for updating the docs!

**Review by chshersh — APPROVED** [review]
Thanks, this looks great! 👏🏻 

I have no suggestions to these changes 🙂 
Could you update your PR and resolve all the conflicts? And I'll merge it 🚢

File: README.md (hunk 1 above)
Comment (zixuanzhang-x) [reply README.md:251]: resolved

**Pull request merged**